Tasks/DownloadPackageV1/package.json

```diff
@@ -21,10 +21,11 @@
     "@types/node": "^16.11.39",
     "azure-devops-node-api": "11.2.0",
     "azure-pipelines-task-lib": "3.3.1",
-    "azure-pipelines-tasks-packaging-common": "^3.244.0",
+    "azure-pipelines-tasks-packaging-common": "^3.246.1",
     "azure-pipelines-tasks-utility-common": "^3.241.0",
     "decompress-zip": "0.3.3",
-    "tar-fs": "1.16.3"
+    "tar-fs": "1.16.3",
+    "extract-zip": "2.0.1"
```

Comment:
I dont think we can add that dependency here -looks like its only applicable to node 20 - see https://github.com/microsoft/azure-pipelines-tasks/commit/95ec9ccde9a8f0a73dc567b0531d7648c2a076fa

Reply:
I double-checked, indeed it only seems to be used in the node20 build.